packages/edit-post/src/components/header/fullscreen-mode-close/index.js

```diff
@@ -12,7 +12,7 @@ import { __ } from '@wordpress/i18n';
 import { addQueryArgs } from '@wordpress/url';
 import { wordpress } from '@wordpress/icons';
 
-function FullscreenModeClose( { showTooltip } ) {
+function FullscreenModeClose( { showTooltip, icon, href } ) {
```

Comment:
Adding these customization options to make it consistent with existing docs and back button in navigation.